tools/podlogs

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+function usage {
+    >&2 echo "Usage:"
+    >&2 echo "google_log_cat [-n numrecords ] <podname>"
+}
+
+if [[ $# -lt 1 ]]
+then
+    usage
+    exit 1
+fi
+
+args=()
+while [[ $# -gt 0 ]]; do
+    case $1 in
+        -n)
+            args+=("--limit")
+            args+=("$2")
+            shift
+            shift
+            ;;
+        *)
+            pod=$1
+            shift
+            ;;
+    esac
+done
+
+gcloud logging read \
+"resource.labels.pod_name=$pod" \
+"${args[@]}" \
+--format='value(jsonPayload.message)' | grep -v '^$' | tac
```

Comment:
Not sure if there's an easy way around it, but dropping the empty lines after the `numrecords` filter often means you get a lot less than `numrecords` printed.

Reply:
Just following up here, I get some surprising behavior pinging a prognostic run's logs:
```
(fv3net) brianh@brian-vm:~/dev/fv3net/tools$ ./podlogs -n 100 restart-prognostic-run-kt97v-3325419850
(fv3net) brianh@brian-vm:~/dev/fv3net/tools$ ./podlogs -n 500 restart-prognostic-run-kt97v-3325419850
(fv3net) brianh@brian-vm:~/dev/fv3net/tools$ ./podlogs -n 700 restart-prognostic-run-kt97v-3325419850
... about 90 lines of logs
(fv3net) brianh@brian-vm:~/dev/fv3net/tools$ ./podlogs -n 1000 restart-prognostic-run-kt97v-3325419850
... about 200 lines of logs
```

Reply:
yah. It's the messages that don't have json.message in them (probably have textPayload).